lib/react_on_rails/locales/base.rb

```diff
@@ -4,7 +4,34 @@
 
 module ReactOnRails
   module Locales
+
+    def self.check_i18n_directory_exists
+      return if ReactOnRails.configuration.i18n_dir.nil?
```

Comment:
I'd prefer to extract `ReactOnRails.configuration.i18n_dir` to a variable or call after https://github.com/shakacode/react_on_rails/blob/21d79dece18547c8b4e709724178a79bcc6ed481/lib/react_on_rails/locales/base.rb#L94 instead of repeating. Same in `self.check_i18n_yml_directory_exists`.